<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterClass</class>
 <widget class="QMainWindow" name="RegisterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nunito Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gartic - Register</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Frontend.qrc">
    <normaloff>:/Frontend/assets/images/appicon.ico</normaloff>:/Frontend/assets/images/appicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-image: url(:/Frontend/assets/images/background.png);
    background-position: center;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>71</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>71</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/gartic-logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_login_from">
        <property name="minimumSize">
         <size>
          <width>261</width>
          <height>405</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>261</width>
          <height>405</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Nunito Medium</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="pushButton_register">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>322</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Nunito Medium</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(255, 191, 0);
color:rgb(0,0,0);
}

QPushButton:hover, QPushButton:pressed {
   background-color: rgb(225, 165, 0);
}</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>221</width>
           <height>58</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_surname">
          <item>
           <widget class="QLabel" name="label_surname">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_surname">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid gray;
</string>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>221</width>
           <height>58</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_password">
          <item>
           <widget class="QLabel" name="label_password">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_password">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid gray;</string>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_register_title">
         <property name="geometry">
          <rect>
           <x>43</x>
           <y>12</y>
           <width>181</width>
           <height>45</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>181</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nunito Medium</family>
           <pointsize>16</pointsize>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/register-badge.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>190</y>
           <width>221</width>
           <height>58</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_username">
          <item>
           <widget class="QLabel" name="label_username">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_username">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid gray;</string>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>221</width>
           <height>58</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_given_name">
          <item>
           <widget class="QLabel" name="label_username_given_name">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Given name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_given_name">
            <property name="font">
             <font>
              <family>Nunito Medium</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid gray;</string>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_goToLogin">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>367</y>
           <width>171</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Nunito Medium</family>
           <pointsize>10</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover, QPushButton:pressed {
	color: rgb(110, 110, 110);
}</string>
         </property>
         <property name="text">
          <string> Already a user? Log In here.</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_surname</tabstop>
  <tabstop>lineEdit_given_name</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_register</tabstop>
  <tabstop>pushButton_goToLogin</tabstop>
 </tabstops>
 <resources>
  <include location="../../Frontend.qrc"/>
  <include location="../../Frontend.qrc"/>
 </resources>
 <connections/>
</ui>
